### PR TITLE
Connect results from api to results to display

### LINKED ISSUE #6: When a photo is taken, the user should be able to click a button to submit the image.

### LINKED ISSUE #8: A user should be able to upload a photo from their camera roll and send it in

### LINKED ISSUE #23: When an image is submitted, a user should receive a response containing possible matches.

### PR DESCRIPTION
**Pull request by Neil-B-Linton**
<h1>PULL REQUEST</h1>

<ul>
  <li>What does it change?</li>
   Connected the Api results to display on the page
  <li>What does it fix?</li>
   Replaced the dummy data with the results from the api call 
</ul>

<h2>Type of Change</h2>

- [x] **Bug Fix**
- [x] **New Feature**
- [x] **New Change**

<h2>How has this been tested?</h2>

- [x] **JS**
- [x] **Dev Tools**
- [x] **Website**

<h2>Checklist</h2>

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new bugs
- [x] I have tagged project partners for review & it has been reviewed 

<h2>Does this PR close any issues?</h2>
close #6 close #8 close #18 close #23

### CODE REVIEW

**Comment by laytonmaes** [comment]
looks good to me.

File: Views/Dashboard.js [hunk 1]
```diff
@@ -41,7 +42,8 @@ export default function Dashboard() {
                             'name_authority',
                             'wiki_description',
                             'taxonomy',
-                            'synonyms'],
+                            'synonyms',
+                            'wiki_image'],
       }
```
Comment (BlueNealis) [inline Views/Dashboard.js:47]: Yay images again

**Review by laytonmaes — APPROVED** [review]

**Review by BlueNealis — APPROVED** [review]
Simple and sweet nice!

**Review by ameliaeiger — APPROVED** [review]
Let's make sure to hook up our display to a state. Otherwise looks good!

**Pull request merged**